### ISSUE TITLE
Enzyme output

### ISSUE THREAD

**Issue by untemps** [Root-App/react-native-mock-render#32]
In my unit tests, I used to render the components using the Enzyme shallow() function. The output printed by debug() looked like a regular tree component:

```
<TouchableOpacity accessibilityComponentType="button">
   <View style={{...}} onLayout={[Function: bound onViewLayout]}>
      <Image source={{...}} style={{...}} />
   </View>
</Component>
```

Using react-native-mock-render and jsdom, the tree nodes are not named anymore:

```
<Component accessibilityComponentType="button">
   <Component style={{...}} onLayout={[Function: bound onViewLayout]}>
      <Image source={{...}} style={{...}} />
   </Component>
</Component>
```

Is it intentional and how can I retrieve each node using the find() function in this case?

**Comment by dabit1** [comment]
I'm running in the same issue. Is it an expected behavior? If it is, for instance, it mean that I can't use find('TextInput') :-1:

**Comment by dabit1** [comment]
@untemps Ey man, I have created a pull request to fix it! #39

**Comment by bob-carson** [comment]
Hi all, thanks for posting this issue!  You should be able to work around this in the short term by finding components by their `testID`.  

I'm happy to help get code changes merged to help address this issue as well. @dabit1, I can reopen that PR for you if you update it to resolve the errors that @mbutsko pointed out.
